deploy/docker/Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV GOOS=linux
 COPY . /go/src/${package}
 RUN go build
 
-FROM alpine:3.18
+FROM alpine:3.20
```

Comment:
Any thoughts on just swapping to `3` ?

Reply:
I am fine with this, it will be built with newest alpine version by default in this case